openstates/sc/people.py

```diff
@@ -50,12 +50,18 @@ def scrape_chamber(self, chamber):
 
             # District # - County - Map
             district = district.split()[1]
-
-            photo_url = leg_doc.xpath('//img[contains(@src,"/members/")]/@src')[0]
-
-            person = Person(name=full_name, district=district,
-                            party=party, primary_org=chamber,
-                            image=photo_url)
+            try:
+                photo_url = leg_doc.xpath('//img[contains(@src,"/members/")]/@src')[0]
+            except IndexError:
+                self.warning("No Photo URL for {}".format(full_name))
+                photo_url = None
```

Comment:
if you set photo_url to '' here, you won't have to special case the constructor below